migrations/schemas/20230410054010-add-table-user-tags.sql

```diff
@@ -0,0 +1,15 @@
+
+-- +migrate Up
+CREATE TABLE IF NOT EXISTS user_tags
+(
+    id UUID PRIMARY KEY NOT NULL DEFAULT uuid_generate_v4(),
+    guild_id integer,
```

Comment:
guild_id and user_id of discord is snowflake so I think we should store them in `string`